docs/coordinates/matchsep.rst

```diff
@@ -73,9 +72,9 @@ rather than the single scalar angular offset of a separation.
 offsets encountered in astronomy.
 
 The first piece of such functionality is the
-:meth:`~astropy.coordinates.SkyCoord.position_angle` method, which gives the
-conventional astronomy position angle (positive angles East of North) from one
-the |skycoord| it is called on to another given as the argument::
+:meth:`~astropy.coordinates.SkyCoord.position_angle` method. This method
+computes the position angle (positive angles East of North) between one
+|skycoord| instance and another (passed as the argument)::
```

Comment:
hmm.  I'm not sure how to re-phrase this better, but I think we want to preserve the idea of "conventional astronomy", by which I mean "the convention used in astronomy".  I think this is important because this is a common mistake (I quite literally just saw it made twice in the last week...) to mix up the convention used in astronomy and other fields like computer science or just regular paper maps...

Maybe 
```suggestion
|skycoord| instance and another (passed as the argument) following the 
astronomy convention (positive angles East of North)::
```
or similar ?